README.md

```diff
@@ -73,4 +73,36 @@ Once the application is running, you will see the message `Deployed successfully
 curl -X POST http://127.0.0.1:8000/video/summarize -Fbody='{"video":{"url":"https://www.youtube.com/watch?v=VhJFyyukAzA"}}'
 ```
 
-See [tutorial](https://mobiusml.github.io/aana_sdk/pages/tutorial/) for more information.
\ No newline at end of file
+See [tutorial](https://mobiusml.github.io/aana_sdk/pages/tutorial/) for more information.
+
+## Running within Docker
+
+To execute the project using Docker, you will find both a Dockerfile and a Docker Compose configuration file in the repository. These resources allow you to run the project seamlessly. You have the option to launch it using Docker Compose for a more streamlined approach, or you can opt to use Docker directly. Here’s how you can do both:
+
+### Deploying via docker-compose
+
+```bash
+CUDA_VISIBLE_DEVICES=0 docker-compose up
+```
+
+The application requires one GPU to operate, so you'll need to set `CUDA_VISIBLE_DEVICES` in the command.
+The docker-compose configuration will deploy a PostgreSQL instance alongside the application and establish a connection between them. If you already have a database set up elsewhere, you can modify the docker-compose configuration by removing PostgreSQL and specifying the PostgreSQL address using environment variables. The application will be accessible at `http://localhost:8000` in the host server and the PostgreSQL will be accessible via port 15430.
+You can check the [docker-compose config](./docker-compose.yaml) to see all available variables you can set for running the application.
+
+### Deploying via docker
+
+To deploy the application by docker directly follow these steps:
+
+1. Build the docker image by running:
+
+```bash
+docker build --no-cache -t aana_summarize_video:latest .
+```
+
+2. Run the the image:
+```bash
+export DB_CONFIG='{"datastore_type":"postgresql","datastore_config":{"host":<PG_HOST>,"port":<PG_PORT>,"user":<PGUSER>,"password":<PG_PASSWORD>,"database":<PG_DB>}}'
```

Comment:
Should we use SQLite as a default option and mention PostgreSQL as an advanced option?

Reply:
Changed